dChatServer/PlayerContainer.h

```diff
@@ -75,5 +77,7 @@ class PlayerContainer {
 	std::map<LWOOBJID, PlayerData*> mPlayers;
 	std::vector<TeamData*> mTeams;
 	std::unordered_map<LWOOBJID, std::u16string> mNames;
+	uint32_t mMaxNumberOfBestFriends = 5;
```

Comment:
Let's at least be consistent now with m_

Reply:
I followed the format of this file, want me to change the rest of them as well?

Reply:
Nah just going forward would be good